modules/ROOT/pages/provisioning-aws.adoc

```diff
@@ -59,3 +59,45 @@ You now should be able to SSH into the instance using the associated IP address.
 ----
 ssh core@<ip address>
 ----
+
+==== Remote ignition configuration
```

Comment:
Let's also remove one level here to let this section appear in the table of content

Reply:
Addressed in #576